app/content/blog/wiki/setting-your-home-server.md

```
@@ -0,0 +1,61 @@
+1. Click on "Aether" the default data center to view selected servers that change the marketplace your queries are run against.
+   ![Screenshot 2023-02-06 231818](https://user-images.githubusercontent.com/124644331/217147247-c706a463-9ce1-4c2f-acd5-032f141d8e42.png)
+
+2. Choose between North American Data Centers, European Data Centers, or Oceanic Data Centers & make world option.
+
+(**NOTE: Each data center offers a variety of World options. Click through each to find your selection.**
+)
+![Screenshot 2023-02-06 232107](https://user-images.githubusercontent.com/124644331/217147575-5d1a14f3-2b16-4f1c-abd3-6837594635d2.png)
+
+![Screenshot 2023-02-06 232553](https://user-images.githubusercontent.com/124644331/217148163-993cb9c4-0fc7-4991-83b3-fa673afcc679.png)
+
+4. Your settings will be automatically updated.
```

markdownlint-cli2 (0.22.1)

[warning] Ordered list item prefix
Expected: 1; Actual: 4; Style: 1/1/1

(MD029, ol-prefix)